sw/device/sca/ecc_serial.c

```diff
@@ -157,33 +203,28 @@ static void p256_ecdsa_sign(const uint32_t *msg, const uint32_t *private_key_d,
                             uint32_t *signature_r, uint32_t *signature_s,
                             const uint32_t *k) {
   uint32_t mode = 1;  // mode 1 => sign
-  LOG_INFO("Copy data");
```

Comment:
I think you don't technically need to remove these `LOG_INFO` calls, since the trigger only actually gets set when `otbn_execute()` is called. You just need to remove the ones after `otbn_execute()`.

Reply:
Yep, you are right. Previously, I put `LOG_INFO` calls to solve an issue. Now, they are kind of useless. In the new version, I converted them into comments for code readability. Would it be okay for you?

Reply:
In the past we noticed that such outputs in the hot paths had a significant effect on throughput. I think it's ok to leave them out unless there is a particular reason we want them.

Reply:
Yep, I agree it's fine to remove them anyway.